.env

```diff
@@ -1,4 +1,10 @@
+DEMO=false
+BROWSERSTACK_USERNAME=user
+BROWSERSTACK_ACCESS_KEY=key
+# Could be all, windows, macMojave, macHighSierra
+BROWSERSTACK_DEVICE_GROUP_NAME=macMojave
```

Comment:
can we move these variable into a new file `.env.test` ? Because it will never be used across the app itself

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables#what-other-env-files-can-be-used

Reply:
ok moved to .env.test